core/templates/dev/head/app.js

```diff
@@ -195,6 +195,15 @@ oppia.factory('oppiaDatetimeFormatter', [function() {
         return date.toLocaleTimeString().replace(/:\d\d /, ' ');
       }
       return date.toLocaleDateString();
+    },
+    // Returns just the date.
+    getLocaleAbbreviatedDateString: function(millisSinceEpoch) {
+      var date = new Date(millisSinceEpoch);
+      if (date.toLocaleDateString() == new Date().toLocaleDateString()) {
```

Comment:
I don't understand -- why not just return date.toLocaleDateString() in all cases?